app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
```

Comment:
@ianroberts131 Consider enabling Devise's `:lockable`, `:timeoutable`, and `:confirmable` modules for added security. 

`:lockable` in particular is worth setting up to mitigate [brute force password guessing attacks](https://www.owasp.org/index.php/Blocking_Brute_Force_Attacks).

You can find out more about each of the modules from the summaries in the bullets near the top of [Devise's README](https://github.com/plataformatec/devise), and following the links next to them to see the API docs which will give you a bit more detail.

If you want to add any or all of these modules to this app feel free, and be sure to add the associated database migrations.